python-bareos/bareos/bsock/lowlevel.py

```diff
@@ -102,7 +102,12 @@ def __init__(self):
         self.max_reconnects = 0
         self.tls_psk_enable = True
         self.tls_psk_require = False
-        self.tls_version = ssl.PROTOCOL_TLS
+        if hasattr(ssl, "PROTOCOL_TLS"):
+            self.tls_version = ssl.PROTOCOL_TLS
+        else:
+            # ssl.PROTOCOL_SSLv23 is deprecated since version Python >= 2.7.13
+            # Use PROTOCOL_TLS instead.
+            self.tls_version = ssl.PROTOCOL_SSLv23
```

Comment:
While this definitely works, I somehow feel that `try:` `except:` would be more in line with general python style.

Reply:
Okay, I've adapted it accordingly.